package.json

```diff
@@ -27,27 +27,34 @@
     "@codecharacter-2022/renderer": "workspace:packages/renderer",
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-brands-svg-icons": "^5.15.4",
+    "@fortawesome/free-regular-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.16",
+    "@fortawesome/react-fontawesome": "^0.1.17",
     "@lit-labs/react": "^1.0.1",
-    "@reduxjs/toolkit": "^1.7.1",
+    "@reduxjs/toolkit": "^1.7.2",
     "@types/grecaptcha": "^3.0.3",
+    "ace-builds": "^1.4.13",
     "bootstrap": "5.1.3",
+    "brace": "^0.11.1",
     "lit": "^2.1.1",
     "phaser": "^3.55.2",
     "phaser3-rex-plugins": "^1.1.66",
     "react": "^17.0.2",
-    "react-bootstrap": "^2.1.1",
+    "react-ace": "^9.5.0",
+    "react-bootstrap": "^2.1.2",
     "react-dom": "^17.0.2",
     "react-flags-select": "^2.1.2",
     "react-google-recaptcha": "^2.1.0",
     "react-hot-toast": "^2.2.0",
     "react-paginate": "^8.1.0",
     "react-redux": "^7.2.6",
+    "react-reflex": "^4.0.3",
```

Comment:
is this needed?

Reply:
Sorry, forgot that one. I'll make all the changes